src/ui/ui_exports/ui_export_defaults.js

```diff
@@ -28,7 +28,7 @@ export const UI_EXPORT_DEFAULTS = {
 
   webpackAliases: {
     ui: resolve(ROOT, 'src/ui/public'),
-    ui_framework: resolve(ROOT, 'ui_framework'),
```

Comment:
__question:__ btw, not related to this PR, do we still need `ui_framework` here https://github.com/elastic/kibana/blob/cb5ee01c6a4c175a7d6681fbabb8e4f86d7b01e5/packages/kbn-eslint-import-resolver-kibana/lib/get_webpack_config.js#L31?